clause/where.go

```diff
@@ -60,6 +60,9 @@ func buildExprs(exprs []Expression, builder Builder, joinCond string) {
 			case Expr:
```

Comment:
We can use case list instead of a separate case:
``` go
case Expr, NamedExpr:
```

Reply:
@sryavuz thanks for the review.

I tried this way first, but it didn't work. Because, for the cases with multiple types, the type of the variable won't be narrowed. So the type of `v` will be Expression interface, which invalidates accessing `SQL` field as `v.SQL`. To handle this, we can use additional type assertion or a `reflection` based approach like `reflect.ValueOf(v).FieldByName("SQL").String()`. However, I think it's better to leave it as is, despite the code duplication.